### PR TITLE
Volcano adapts to the k8s v1.30

### PR DESCRIPTION
**Pull request by wangyysde**
fix https://github.com/volcano-sh/volcano/issues/3523
The volcano api need to adapts to the k8s v1.30。
The pr of volcano adapts to the k8s v1.29 :https://github.com/volcano-sh/volcano/pull/3534

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -1,13 +1,13 @@
 module volcano.sh/apis
 
-go 1.21
+go 1.22
```
Comment (Monokaix) [inline go.mod:3]: Does v1.30 need go version 1.22?

Comment (wangyysde) [reply go.mod:3]: I have refer to: https://github.com/kubernetes/kubernetes/blob/release-1.30/go.mod 
So I think that k8s v1.30 using go version 1.22. Is it right?

**Comment by guoqinwill** [comment]
Is the go.sum file not updated?

**Comment by wangyysde** [comment]
> Is the go.sum file not updated?

I have run go mod tidy just now. I think that this operation updates go.sum file. Is it right?

File: go.mod [hunk 2]
```diff
@@ -1,15 +1,17 @@
 module volcano.sh/apis
 
-go 1.21
+go 1.22.0
+
+toolchain go1.22.2
```
Comment (Monokaix) [inline go.mod:5]: I think we can remove this: )

Comment (wangyysde) [reply go.mod]: I have modified it.

**Comment by guoqinwill** [comment]
> > Is the go.sum file not updated?
> 
> I have run go mod tidy just now. I think that this operation updates go.sum file. Is it right?

Yes. And please combine these commits into one.

**Comment by wangyysde** [comment]
> > Is the go.sum file not updated?
> 
> I have run go mod tidy just now. I think that this operation updates go.sum file. Is it right?

Ok , I have squash the commits

**Comment by guoqinwill** [comment]
This PR https://github.com/volcano-sh/apis/pull/129, which updates the Kubernetes version based on your PR, has been merged. Your PR can now be closed. Thank you very much for your contribution.

**Pull request closed without merging**